fast64_internal/oot/oot_level_parser.py

```diff
@@ -727,12 +727,15 @@ def parseTransActorList(
 
             sharedSceneData.transDict[actorHash] = actorObj
 
-            if roomIndexFront != 255:
-                parentObject(roomObjs[roomIndexFront], actorObj)
-                transActorProp.roomIndex = roomIndexBack
+            fromRoom = roomObjs[roomIndexFront]
+            toRoom = roomObjs[roomIndexBack]
+            if roomIndexFront != roomIndexBack:
```

Comment:
The old logic for checking for 255 is gone, are you sure this is equivalent? (e.g. `roomIndexFront == validRoomIndex` and `roomIndexBack == 255` is not possible?) I don't know this system so I don't know offhand.

Reply:
iirc 255 is only used for non-transition doors, idr the specifics though but I do remember noticing a weirdness like this

Reply:
I'll take a look at this to figure it out

Reply:
(for the record: I remembered the reason and it's better to check if the room is the same on both sides to tell if a room transitions to another one or not since the camera can be anything you want)